sklearn/cluster/_mean_shift.py

```diff
@@ -38,20 +38,22 @@ def estimate_bandwidth(X, quantile=0.3, n_samples=None, random_state=0,
     X : array-like of shape (n_samples, n_features)
         Input points.
 
-    quantile : float, default 0.3
+    quantile : float, default=0.3
         should be between [0, 1]
         0.5 means that the median of all pairwise distances is used.
+        The default is 0.3
 
-    n_samples : int, optional
+    n_samples : int, default=None
         The number of samples to use. If not given, all samples are used.
+        The default value is None.
 
     random_state : int, RandomState instance, default=None
         The generator used to randomly select the samples from input points
         for bandwidth estimation. Use an int to make the randomness
-        deterministic.
+        deterministic.The default value is None.
```

Comment:
no need either, here and in the other places

Reply:
I've removed all the defaults I'd added in the descriptions.